src/calculateRank.js

```diff
@@ -13,12 +31,24 @@ function calculateRank({
   const STARS_OFFSET = 0.75;
   const PRS_OFFSET = 0.5;
   const FOLLOWERS_OFFSET = 0.45;
+  const REPO_OFFSET = 1;
+
+  const ALL_OFFSETS =
+    CONTRIBS_OFFSET +
+    ISSUES_OFFSET +
+    STARS_OFFSET +
+    PRS_OFFSET +
+    FOLLOWERS_OFFSET +
+    REPO_OFFSET;
 
-  const FIRST_STEP = 0;
-  const SECOND_STEP = 5;
-  const THIRD_STEP = 20;
-  const FOURTH_STEP = 50;
-  const FIFTH_STEP = 130;
+  const RANK_S_VALUE = 1;
+  const RANK_DOUBLE_A_VALUE = 25;
+  const RANK_A2_VALUE = 45;
+  const RANK_A3_VALUE = 60;
+  const RANK_B_VALUE = 100;
+
+  const TOTAL_VALUES =
+    RANK_S_VALUE + RANK_A2_VALUE + RANK_A3_VALUE + RANK_B_VALUE;
```

Comment:
Adding all the OFFSETS & conditional values and feeding it to that function, all values are result of trial and errors

Reply:
@anuraghazra Hi, I wanted to know , what are these offsets are for and how are they calculated??/